sites/website/src/docs/design-systems/overview.md

```diff
@@ -5,7 +5,7 @@ sidebar_label: Overview
 custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/design-systems/overview.md
 ---
 
-A Design System can generally be thought of as a collection of resources for interactive media that promotes brand alignment. While that definition is intentionally broad, in UI development, Design Systems generally manifest as component libraries surrounded by usage guidance and design principles.
+A Design System is a collection of interactive media resources that promotes brand alignment. While that definition is intentionally broad, in UI development, Design Systems manifest as component libraries surrounded by usage guidance and design principles.
```

Comment:
I think the nuance above is preferred here as opposed to more definitive language; there is nuance around design systems and I'm not sure I'd define it explicitly as we have above with `is`.

Reply:
For conciseness, how about a compromise, @chrisdholt, where I replace "is" with "is generally"?

Reply:
Sure, I think as long as it's not an absolute statement that's fine. The nuance seems to still be there with `is generally`